src/less/combobox/base/combobox.less

```diff
@@ -1,40 +1,71 @@
+
 .combobox {
     box-sizing: border-box;
     display: inline-block;
 }
 
 .combobox__control > input {
     appearance: none;
+    background-color: @combobox-input-background-color;
     border: 1px solid;
```

Comment:
Oh, one more nit, sorry missed it last time. 
Can add to this attribute  instead of having a different border-color attribute.
border: 1px solid @combobox-input-border-color;

Reply:
Doh! Good catch. I'll fix it.